client/http/client.go

```diff
@@ -239,6 +267,8 @@ func (c *client) request(
 	logFields := []zap.Field{
 		zap.String("name", name),
 		zap.String("url", url),
+		zap.String("method", method),
+		zap.String("caller-id", c.callerID),
```

Comment:
Do we log the caller id on the PD server side?

Reply:
It will appear in the audit log also.